Tests/JWTKitTests/ECDSATests.swift

```diff
@@ -3,26 +3,27 @@ import BigInt
 import XCTest
 
 final class ECDSATests: XCTestCase {
-    func testECDSADocs() throws {
-        let signers = JWTSigners()
-        XCTAssertNoThrow(try signers.use(.es256(key: .public(pem: ecdsaPublicKey))))
+    func testECDSADocs() async throws {
+        XCTAssertNoThrow(try ES256Key.public(pem: ecdsaPublicKey))
```

Comment:
Shouldn't this test try adding the key to the collection

Reply:
Adding a key to the collection doesn't throw